src/components/themed/WalletListCurrencyRow.tsx

```diff
@@ -50,13 +55,40 @@ const WalletListCurrencyRowComponent = (props: Props) => {
 
   return (
     <TouchableOpacity accessible={false} style={styles.row} onLongPress={handleLongPress} onPress={handlePress}>
-      {customAsset != null ? <CustomAssetRow customAsset={customAsset} /> : <CurrencyRow showRate={showRate} token={token} tokenId={tokenId} wallet={wallet} />}
+      {isPaused ? (
+        <>
+          <View style={styles.overlayLabelContainer}>
+            <EdgeText style={styles.overlayLabel}>{lstrings.fragment_wallets_wallet_paused}</EdgeText>
+          </View>
+          <View style={styles.overlayContainer} />
+        </>
+      ) : null}
+      {customAsset != null ? (
+        <CustomAssetRow customAsset={customAsset} />
+      ) : (
+        <CurrencyRow showRate={showRate && !isPaused} token={token} tokenId={tokenId} wallet={wallet} />
+      )}
     </TouchableOpacity>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  // Layout:
+  overlayLabel: {
+    color: theme.overlayDisabledTextColor
```

Comment:
Though default text color in edgeDark is white, I want to force this to be white no
matter what so it contrasts against the background which will always be on the
darker side